client/package.json

```diff
@@ -6,8 +6,9 @@
     "@material-ui/core": "^4.11.0",
     "@material-ui/data-grid": "^4.0.0-alpha.8",
     "@material-ui/icons": "^4.9.1",
-    "clsx": "^1.1.1",
     "axios": "^0.21.0",
+    "clsx": "^1.1.1",
+    "moment": "^2.29.1",
```

Comment:
moment has a lot of [unnecessary overhead + weird quirks](https://github.com/you-dont-need/You-Dont-Need-Momentjs) and it got [deprecated](https://github.com/moment/moment/blob/develop/README.md) last month, so maybe [dayjs](https://github.com/iamkun/dayjs) (extremely lightweight but might need to install extra plugins) or [date-fns](https://github.com/date-fns/date-fns) (lightweight modular utility lib, more of a 1:1 to moment) might be better libraries to use for date parsing!

Reply:
Ah crap, totally missed that part about it being deprecated. This was the most popular date time library for the longest time, wtf happened!!

Reply:
@resurreccionl It looks like dayjs has the same exact api, at least for the functions I'm currently using, I'll switch to it tomorrow.

Reply:
i thought it was totally out of the blue too LOL. i think the maintainers just never anticipated how common web app bundles would be and now its super bulky for most projects